src/coreComponents/constitutive/ConstitutiveBase.cpp

```diff
@@ -53,7 +53,7 @@ void ConstitutiveBase::allocateConstitutiveData( Group & parent, localIndex cons
       if( wrapper.second->sizedFromParent() )
       {
         string const wrapperName = makeFieldName( this->getName(), wrapper.first );
-        parent.registerWrapper( wrapper.second->clone( wrapperName, parent ) ).
+        parent.registerWrapper( wrapper.second->clone( wrapperName, parent ), true ).
```

Comment:
These sorts of changes `registerWrapper(..., true)` and `registerGroup(..., true)` permit an object with the given name to already exist, and return the existing object.